src/SmartTransactionsController.ts

```diff
@@ -477,8 +566,9 @@ export default class SmartTransactionsController extends BaseController<
   async getFees(
     tradeTx: UnsignedTransaction,
     approvalTx: UnsignedTransaction,
+    networkClientId?: NetworkClientId,
```

Comment:
Do we want to put this in an options object too?

Reply:
how do I keep missing these 😮‍💨

Reply:
There's a lot in this one!

Reply:
done here: https://github.com/MetaMask/smart-transactions-controller/pull/210/commits/99fc2cc69f1b6a520d23b3eb77f092c883305a78